biometric-auth/build.gradle

```
@@ -3,7 +3,7 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion 28
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 16
```

As fingerprint support added in Android M (API 23) and biometric API changed in API level 28, `minSdkVersion` could be changed to 23.
```suggestion
        minSdkVersion 23
```